mutmut/__main__.py

```diff
@@ -187,12 +195,18 @@ def climain(command, argument, paths_to_mutate, backup, runner, tests_dir,
     show [mutation id]\n
         Show a mutation diff.\n
     """
-    sys.exit(main(command, argument, paths_to_mutate, backup, runner, tests_dir,
-            swallow_output, use_coverage, dict_synonyms, cache_only, version,
-            suspicious_policy, untested_policy))
+    if test_time_base is None:  # click sets the default=0.0 to None
```

Comment:
If you provide `0` or `0.0` as an argument to either `--test-time-multiplier` or `--test-time-base` `click` will interpret this as `None`. This resolves this odd behavior.